EFCore.CheckConstraints.Test/EnumCheckConstraintTest.cs

```diff
@@ -136,12 +157,91 @@ public void TPT()
 
     #region Support
 
-    private enum CustomerType
+    // ensure int is used explicitly
```

Comment:
I don't think being explicit is necessary... The fact that enums are implicitly backed by int is part of the language spec, so we can assume that.